kdumpctl

```diff
@@ -650,6 +662,9 @@ load_kdump()
 
 	KEXEC_ARGS=$(prepare_kexec_args "${KEXEC_ARGS}")
 	KDUMP_COMMANDLINE=$(prepare_cmdline "${KDUMP_COMMANDLINE}" "${KDUMP_COMMANDLINE_REMOVE}" "${KDUMP_COMMANDLINE_APPEND}")
+	if is_aws_aarch64; then
+		KDUMP_COMMANDLINE=$(echo ${KDUMP_COMMANDLINE} | sed 's/irqpoll//' )
```

Comment:
Hi @pfliu since this is a workaround, could you also leave a code comment in case someone else changes this part of code in the future?